Training/1.0 Introduction.md

```diff
@@ -12,37 +12,65 @@ First we we learn a little bit about coding in Python, although many of you will
 List all the Data Types of Python with examples.  
 
 ```
-
+# int
+a = 100
+# float
+b = 1.2
+# string
+c = 'acc'
 ```
 
 ### Task 2: 
 
 Other useful ways of storing data are in List, Arrays and Tuples. Create one of each and assign them to the variables a, b and c respectively. 
 
 ```
-
+# list
+a = [1, 2, 3, 4]
+# tuple
+c = (1, 2, 3, 4)
+# dictionary
+d = {1:"first name",2:"last name", "age":33}
+# arrarys
+import numpy as np
+b = np.array([[[4,36], [134, 94]], [[976, 234], [189, 123]],[[56,21],[109,67]]])
 ```
 
 ### Task 3: 
 
 We are also interested in becoming good software developers so we will need to use conditional, loops. Write an statement where the summation of a variable x+1 will be calculated if the value of x is greater than 2. 
 
 ```
-
+x = int(input ("input a integer"))
+if x > 2:
+    x += 1
 ```
 
 Now incorporate a for loop to calculate the value of n(x+1) for n repetitions where n=3. Store each value of the for loop in an array named y.  
 
 ```
-
+# I don't know which series of n I should use(n = 0 and 1 and 2 or n = 1 and 2 and 3) so I decided to use n = 0 then 1 then 2 like this: 
+y = np.zeros(3)
+for n in range(3):
+    y[n] = n * (x + 1)
 ```
 
 ### Task 4: 
 
 Finally we will have a look at creating functions. Functions allow us to compute processes much faster without having to repeat lines of code. Using your code from Task 3, create a function called 'my_cool_function', which takes in the value x and will compute the n(x+1) if x is greater than 2 and for n repetitions where values will be stored in an array named y. Your function will return y. 
 
 ```
-
+n = int(input("input n iterations"))
+x = int(input ("input a integer"))
+y = np.zeros(n)
+def my_cool_function(x):
+    if x > 2:
+        for i in range(n):
+            y[i] = i * (x + 1)
+    print(y)
+      
+my_cool_function(x)
+    
```

Comment:
Correct, well done! Task says you should return y not print y, but everything else is perfect.